prime-dotnet-webapi/Services/CommunitySiteService.cs

```diff
@@ -244,38 +281,112 @@ private void UpdateContacts(CommunitySite current, CommunitySiteUpdateModel upda
                     }
                 }
             }
+            return result;
         }
 
-        private void UpdateBusinessHours(Site current, CommunitySiteUpdateModel updated)
+        private static string TranslateContactType(string contactType)
         {
+            return contactType switch
+            {
+                "AdministratorPharmaNet" => "Administrator PharmaNet",
+                "PrivacyOfficer" => "Privacy Officer",
+                "TechnicalSupport" => "Technical Support",
+                _ => ""
+            };
+        }
+
+        private string CompareContact(Contact currentContact, Contact newContact)
+        {
+            var result = new List<string>();
+            var propertyNames = new[,]
+            {
+                {nameof(currentContact.FirstName), "First Name"},
+                {nameof(currentContact.LastName), "Last Name"},
+                {nameof(currentContact.JobRoleTitle), "Job Title"},
+                {nameof(currentContact.Email), "Email"},
+                {nameof(currentContact.Phone), "Phone"},
+                {nameof(currentContact.Fax), "Fax"},
+                {nameof(currentContact.SMSPhone), "SMS Phone"},
```

Comment:
For some reason, my change to SMS Phone was not captured in the Event Log

Reply:
It is fixed.